src/app/card/info-status-card/examples/info-status-card-example.component.html

```diff
@@ -1,57 +1,47 @@
-<div class="padding-15">
-  <div class="row">
-    <div class="col-xs-12">
-      <h4>Info Card Component Example</h4>
-      <hr/>
-    </div>
+<div class="row">
+  <div class="col-xs-12">
+    <h4>Info Card Component Example</h4>
+    <hr/>
   </div>
-  <div class="cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-sm-4">
+</div>
+<div class="padding-15">
+  <div class="row row-cards-pf well">
+    <div style="max-width:409px;">
+      <div class="col-md-10">
         <label>With top border, Icon Class, Href</label>
         <pfng-info-status-card
           [config]="card1Config">
         </pfng-info-status-card>
-      </div>
-    </div>
-  </div>
-  <div class="cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-sm-4">
+        <br>
```

Comment:
Instead of using `<br>`, you can make each card it's own row. That will give proper spacing and allow for responsiveness on resizing of the viewport.